app.js

```diff
@@ -0,0 +1,129 @@
+var path = require('path');
+var app = express();
+var server = require('http').Server(app);
+var bodyParser = require('body-parser');
+var session= require('express-session');
+var multer = require('multer');
+var mysql = require('mysql');
+var con = mysql.createConnection({
+    host: "localhost",
+    user: "root",
+    password: "",
+    database: "ted"
+});
+con.connect(function (err) {
+    if (err) throw err;
+    console.log("Connected!");
+});
+app.use(express.static(path.join(__dirname, 'www')));
+app.use(bodyParser.json() );
+app.use(bodyParser.urlencoded({ extended: true}));
+app.use(session({
+    secret: 'ted-x-gawds',
+    resave: true,
+    saveUninitialized: true
+}));
+var Storage = multer.diskStorage({
+    destination: function(req, file, callback) {
+        callback(null, "./www/images");
+    },
+    filename: function(req, file, callback) {
+        callback(null, file.fieldname + "_" + Date.now() + "_" + file.originalname);
+    }
+});
+var upload = multer({
+    storage: Storage
+}).array("images", 3);
+app.get('/login' , function (req , res) {
+    if(req.session.user) {
+        res.redirect('/admin');
+    }else {
+        res.sendFile(__dirname + '/www/login.html');
+    }
+});
+app.post('/login' , function (req , res) {
+    login(req , res);
+});
+app.get('/admin'  ,  function (req , res) {
+    console.log(req.session.user);
+    if(!req.session.user) {
+        console.log("Admin");
+        res.redirect('/login');
+    }else {
+        res.sendFile(__dirname + '/www/admin.html');
+    }
+});
+app.get('/speaker' , function (req , res) { 
+	res.sendFile(__dirname + '/www/speaker.html');
+});
+app.post('/speaker' , function (req , res) { 
+    var sql = "SELECT * FROM speaker ";
+    con.query(sql, function (err, result, fields) {
+        var jsonString = JSON.stringify(result);
+        var jsonData = JSON.parse(jsonString);
+        res.send(jsonData);
+    });
+});
+app.post("/speakerinsert", function(req, res) {
+    upload(req ,res, function(err) {
+        if (err) {
+            return res.end("Something went wrong!");
+        }
+        return res.end("File uploaded sucessfully!.");
+    });
+    insertSpeaker(req , res);
+});
+app.get('/logout', function (req, res) {
+    req.session.destroy();
+    res.redirect("/login");
+});
+app.post('/getuser' , function (req , res) {
+    res.send({"username" : req.session.user});
+});
+app.get('/blog' , function (req , res) { 
+	res.sendFile(__dirname + '/www/blog.html');
+});
+app.post('/blog' , function (req , res) { 
+    var sql = "SELECT * FROM videos";
+    con.query(sql, function (err, result, fields) {
+        var jsonString = JSON.stringify(result);
+        var jsonData = JSON.parse(jsonString);
+        res.send(jsonData);
+    });
+});
+function login(req,res){
+    var post = req.body;
+    var username  = post.user;
+    var password = post.password;
+    var sql = "SELECT * FROM login WHERE username='"+username+"'";
```

Comment:
We should play safe here, you must have heard of SQL Injection. @rewanth1997 knows better.
Let's sanitize user input before sending it to mysql.